### PR TITLE
23.2.1 - PB-3588

### PR DESCRIPTION
**Pull request by sgajawada-px**
**What type of PR is this?**
> Uncomment only one and also add the corresponding label in the PR:
bug
>feature
>improvement
>cleanup
>api-change
>design
>documentation
>failing-test
>unit-test
>integration-test

**What this PR does / why we need it**:
Here while listing the namespace we were using the sched-ops  ListNamespacesV2 to accept the label selector as a string.

**Does this PR change a user-facing CRD or CLI?**:
Yes in ApplicationBackupSpec we are changing the NsLabelSelector type from a mapper to string
Because if the labelselector has only key then the mapper doesnt support this.

**Is a release note needed?**:

```release-note
Issue:
User Impact:
Resolution

```

**Does this change need to be cherry-picked to a release branch?**:

Note: This PR is raised only for the code review. unit testing is currently in-progress will post status once it is DONE.

### CODE REVIEW

File: pkg/apis/stork/v1alpha1/applicationbackup.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ type ApplicationBackupSpec struct {
 	PlatformCredential string                             `json:"platformCredential"`
 	RancherProjects    map[string]string                  `json:"rancherProjects"`
 	Selectors          map[string]string                  `json:"selectors"`
-	NamespaceSelector  map[string]string                  `json:"namespaceSelector"`
+	NamespaceSelector  string                             `json:"namespaceSelector"`
```
Comment (lalat-das) [inline pkg/apis/stork/v1alpha1/applicationbackup.go:36]: Remove trailing space here

**Review by lalat-das — APPROVED** [review]
I guess the pkg/client/*  are all generated because of the **codegen**  binary run.. auto generated code right ? They look very unrelated hence confused...

**Comment by lalat-das** [comment]
Fill the PR template appropriately every-time. And kindly post the unit test screenshot  at the end of the template.

**Comment by lalat-das** [comment]
Because too many branches are involved, Please mention in vendor commit from which branch of which repo this vendoring is happening ..   Please update the commit message accordingly.  sched-ops master to stork 23.2.1 etc..

File: pkg/apis/stork/v1alpha1/applicationbackup.go (hunk 1 above)
Comment (sgajawada-px) [reply pkg/apis/stork/v1alpha1/applicationbackup.go:36]: This trailing space is due to the go fmt

**Pull request closed without merging**